Automattic/wp-calypso
# Gutenboarding: remove unused static files

## Changes proposed in this Pull Request

☢️ ~D48054-code removes some unused endpoints, and should be deployed first as it refers to paths deleted by this PR!~ ✅ 

Hey, we're banishing unused template-related files to that place on the Internet where JSON files go to retire, whereby "retire" is a euphemism for uselessly whittling out eternity in a data center somewhere.

We're also relocating design screenshots to live under `/images`

# Testing instructions

Apply D48054-code (if it hasn't been deployed already)

1. Fire up http://calypso.localhost:3000/new
2. Check that the design screenshots still show (in both Chrome and Safari to make sure both webp/jpg files work)
3. Check the previews (this ensure that the demo endpoint is still working)
4. Create a couple of site. All good?
5. Take a few deep breaths and marvel at the size of the cosmos
